core/src/main/resources/hudson/model/Run/confirmDelete.jelly

```diff
@@ -28,6 +28,7 @@ THE SOFTWARE.
          xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
   <l:layout title="${it.fullDisplayName}">
     <st:include page="sidepanel.jelly"/>
+    <l:breadcrumb title="${%delete.build(it.displayName)}" />
```

Comment:
`Delete the build ‘#7’?` doesn't seem appropriate to me.
As @uhafner has mentioned elsewhere these sort of pages ideally wouldn't exist and would be a modal, I don't think a breadcrumb entry is great for this but what about?
```suggestion
    <l:breadcrumb title="${%Delete build}" />
```

or

```suggestion
    <l:breadcrumb title="${%Confirm deletion}" />
```